planet/scripts/v1.py

```diff
@@ -408,25 +408,28 @@ def list_feeds(pretty, limit, stats):
 @feeds.command('list-mosaics')
 @click.argument('feed_id')
 def get_mosaic_list_for_feed(feed_id):
-    '''List mosaics linked to feed'''
+    """List mosaics linked to feed."""
     cl = clientv1()
     feed_info = cl.get_feed_info(feed_id).get()
 
-    for type_ in ['target', 'source']:
-        feed_image_conf = feed_info.get(type_)
+    for type_ in ["target", "source"]:
+        click.echo("{} mosaics:".format(type_))
```

Comment:
I want to point out that injecting headers into a program's output makes the output less consumable by other programs. For future versions we might want to print lines like

```json
{"type": "source", "name": "yadayada"}
{"type": "target", "name": "badabing"}
```

so that we can filter with jq or whatever.

cc @jreiberkyle